plugins/plugin-chart-echarts/src/Pie/buildQuery.ts

```diff
@@ -19,5 +19,11 @@
 import { buildQueryContext, QueryFormData } from '@superset-ui/core';
 
 export default function buildQuery(formData: QueryFormData) {
-  return buildQueryContext(formData);
+  const { metric } = formData;
+  return buildQueryContext(formData, baseQueryObject => [
+    {
+      ...baseQueryObject,
+      orderby: [[metric, false]],
```

Comment:
This is now needed to do descending ordering by the chosen metric: https://github.com/apache/incubator-superset/pull/11153